api/model.go

```diff
@@ -176,10 +176,11 @@ type Event struct {
 
 type SupportBundle struct {
 	client.Resource
-	NodeID       string                     `json:"nodeID"`
-	State        manager.BundleState        `json:"state"`
-	Name         string                     `json:"name"`
-	ErrorMessage manager.BundleErrorMessage `json:"errorMessage"`
+	NodeID                     string                     `json:"nodeID"`
+	State                      manager.BundleState        `json:"state"`
+	Name                       string                     `json:"name"`
+	ErrorMessage               manager.BundleErrorMessage `json:"errorMessage"`
+	DownloadProgressPercentage float64                    `json:"downloadProgressPercent"`
```

Comment:
`Progress string `json:"progress"` is enough.

And don't put float in the json.